tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -0,0 +1,368 @@
+    def test_positive_provision_vmware_with_image(self):
+        """ Provision a host on vmware compute resource with image based
+
+        @id: 2cbddac9-c5fa-4f6e-a098-d3e47a3aeb3c
+
+        @setup: vmware hostname and credentials.
+
+        1. Configured subnet for provisioning of the host.
+        2. Configured domains for the host.
+        3. Population of images into satellite from vmware templates.
+        4. Activation key and CV for the host.
+
+        @steps:
+
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select vmware compute resource from "Deploy on" drop down.
+        4. Associate appropriate feature capsules.
+        5. Go to "operating system tab".
+        6. Edit Provisioning Method to image based.
+        7. Select the appropriate image .
+        8. Associate the activation key and submit.
+
+        @Assert: The host should be provisioned successfully
+
+        @Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_vmware_with_compute_profile(self):
+        """ Provision a host on vmware compute resource with compute profile
+        default (3-Large)
+
+        @id: cfe68708-f062-425e-bed7-a46e04007b11
+
+        @setup: vmware hostname ,credentials and provisioning setup.
+
+        @steps:
+
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select vmware compute resource from "Deploy on" drop down.
+        4. Select the "Compute profile" from the drop down.
+        5. Provision the host using the compute profile.
+
+        @Assert: The host should be provisioned successfully
+
+        @Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_vmware_with_custom_compute_settings(self):
+        """ Provision a host on vmware compute resource with
+         custom disk, cpu count and memory
+
+        @id: d82c2b81-3a24-4d6e-82eb-c35709861a44
+
+        @setup: vmware hostname ,credentials and provisioning setup.
+
+        @steps:
+
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select vmware custom compute resource from "Deploy on" drop down.
+        4. Select the custom compute profile" with custom disk size, cpu count
+           and memory.
+        5. Provision the host using the compute profile.
+
+        @Assert: The host should be provisioned with custom settings
+
+        @Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_vmware_with_host_group(self):
+        """ Provision a host on vmware compute resource with
+        the help of hostgroup.
+
+        @id: d4e442ad-77f1-4d5e-9d1b-9a60d69b034f
+
+        @setup: vmware hostname ,credentials provisioning setup and hostgroup
```

Comment:
@setup:

1. Setup CR with vmware hostname and credentials.
2. Configure provisioning setup.
3. Make sure a hostgroup is created and all required entities should be associated.